public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -343,7 +370,7 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
         {isLoading || isLoadingAnomalyResults ? (
           <EuiFlexGroup
             justifyContent="spaceAround"
-            style={{ height: '200px', paddingTop: '100px' }}
+            style={{ height: '500px', paddingTop: '100px' }}
```

Comment:
Why change to 500px here ? Will it impact realtime detector?

Reply:
This change was to prevent the page to look as jerky when in the loading state, since it would often cause the screen to add & remove the scrollbar. It does affect the realtime detector, but improves that as well since it is just an aesthetic improvement.

Reply:
Got it, will it impact the realtime detector's AD result chart?

Reply:
Yes, please see above comment. It's an improvement for both real-time and historical.